code/modules/admin/verbs/randomverbs.dm

```diff
@@ -26,12 +25,11 @@
 
 	if(!ismob(M))
 		return
-	if (!holder)
-		to_chat(src, "Only administrators may use this command.")
+	if(!check_rights(R_ADMIN))
 		return
 
 	message_admins("[key_name_admin(src)] has started answering [key_name(M.key, 0, 0)]'s prayer.")
-	var/msg = input("Message:", text("Subtle PM to [M.key]")) as text
+	var/msg = input("Message:", text("Subtle PM to [M.key]")) as text|null

```

Comment:
I am not sure if these need a |null or if it does have a cancel button already. I just added it because num|null is needed for a cancel button for input.